src/Models/Interfaces/IEmail.ts

```diff
@@ -0,0 +1,19 @@
+import { Model } from '../Model';
+
+export default interface IEmail {
+    email: string;
+}
+
+export class Email extends Model implements IEmail {
+    constructor(data: IEmail) {
+        super(data);
+    }
+
+    get email() {
+        return '';
+    }
+
```

Comment:
Same reason as the setter only if the value of the parmeter that is sent is undefined we use the value of the getter as the default otherwise it just remains undefined